src/pyOpenMS/pxds/IdentificationDataConverter.pxd

```diff
@@ -11,9 +11,10 @@ cdef extern from "<OpenMS/METADATA/ID/IdentificationDataConverter.h>" namespace
     
     cdef cppclass IdentificationDataConverter "OpenMS::IdentificationDataConverter":
         IdentificationDataConverter() nogil except +
-        IdentificationDataConverter(IdentificationDataConverter &) nogil except +
-        void importIDs(IdentificationData & id_data, const libcpp_vector[ ProteinIdentification ] & proteins, const libcpp_vector[ PeptideIdentification ] & peptides) nogil except +
-        void exportIDs(const IdentificationData & id_data, libcpp_vector[ ProteinIdentification ] & proteins, libcpp_vector[ PeptideIdentification ] & peptides, bool export_oligonucleotides) nogil except +
-        MzTab exportMzTab(const IdentificationData & id_data) nogil except +
+        IdentificationDataConverter(IdentificationDataConverter &) nogil except + #wrap-ignore
```

Comment:
```suggestion
        IdentificationDataConverter(IdentificationDataConverter &) nogil except +
```

Reply:
Again, please be careful when merging the release branch. 
If you are not sure, please check here:
https://github.com/OpenMS/OpenMS/pull/5460/files

Reply:
Noted. Thankyou @oliveralka.